tests/components/blebox/test_switch.py

```diff
@@ -0,0 +1,386 @@
+"""Blebox switch tests."""
+
+import logging
+
+import blebox_uniapi
+import pytest
+
+from homeassistant.components.switch import DEVICE_CLASS_SWITCH
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_OFF,
+    STATE_ON,
+)
+
+from .conftest import (
+    async_setup_entities,
+    async_setup_entity,
+    mock_feature,
+    mock_only_feature,
+    setup_product_mock,
+)
+
+from tests.async_mock import AsyncMock, PropertyMock
+
+
+@pytest.fixture(name="switchbox")
+def switchbox_fixture():
+    """Return a default switchBox switch entity mock."""
+    feature = mock_feature(
+        "switches",
+        blebox_uniapi.switch.Switch,
+        unique_id="BleBox-switchBox-1afe34e750b8-0.relay",
+        full_name="switchBox-0.relay",
+        device_class="relay",
+        is_on=False,
+    )
+    feature.async_update = AsyncMock()
+    product = feature.product
+    type(product).name = PropertyMock(return_value="My switch box")
+    type(product).model = PropertyMock(return_value="switchBox")
+    return (feature, "switch.switchbox_0_relay")
+
+
+async def test_switchbox_init(switchbox, hass, config):
+    """Test switch default state."""
+
+    feature_mock, entity_id = switchbox
+
+    feature_mock.async_update = AsyncMock()
+    entry = await async_setup_entity(hass, config, entity_id)
+    assert entry.unique_id == "BleBox-switchBox-1afe34e750b8-0.relay"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5535891)